src/main/java/com/marklogic/appdeployer/AppConfig.java

```diff
@@ -199,6 +199,12 @@
 	private Pattern resourceFilenamesExcludePattern;
 	private Pattern resourceFilenamesIncludePattern;
 
+	//Fields to exclude from JSON payloads
+	private String[] excludeFields;
+	//Fields to include in JSON Payloads and its values
+	// example: triple-index,true
+	private Map<String, String> includeFields;
```

Comment:
I may have mistakenly implied this as a Map - I think it should just be a String[] as well. The use case would be - I've got this big database.json file, but I only want to e.g. apply the triple-index field when the database is updated.